apps/admin-gui/src/app/shared/components/one-entity-attribute-page/one-entity-attribute-page.component.ts

```diff
@@ -85,6 +85,12 @@ export class OneEntityAttributePageComponent implements OnInit {
           this.loading = false;
         });
         break;
+      case 'host':
+        this.attributesManagerService.getHostAttributes(this.entityId).subscribe((attributes) => {
+          this.attributes = attributes;
```

Comment:
I think that there should be also this according to the old behavior.
```suggestion
          this.attributes = filterCoreAttributes(attributes);
```

Reply:
EDIT: `filterCoreAttributes` is called in attribute-list, so there it would be redundant